packages/webcert/src/components/header/WebcertHeaderUnit.tsx

```diff
@@ -24,25 +36,104 @@ const InactiveUnit = styled.span`
   font-size: 12px;
 `
 
+const StyledButton = styled.button`
+  border: none;
+`
+
+const Link = styled.button`
+  text-decoration: none;
+`
+
+const ExpandedDiv = styled.div`
```

Comment:
Style should be placed somewhere where it can be reused

Reply:
The style has been moved to ExpandableBox.